migration/validator-registration.md

```diff
@@ -0,0 +1,95 @@
+
+## Validator Registration Tool
+
+This guide focuses on **Phase 1: Validator Registration** of the migration process, which takes place between **September 12 and October 6**. During this phase, you will create and register your validator on the PoW Mainnet, preparing it for the transition to PoS.
+
+Nimiq has developed a **Validator Registration Tool** specifically for this process. By using this tool, validators pre-register within the PoW chain, ensuring they are ready for the PoS transition. The tool operates in two modes:
+
+- **Key and Address Generation**: Generates the validator's keys and corresponding address
+- **Validator Registration**: Registers the validator by sending the necessary transactions on the PoW chain
+
+| **Phase** | **Start** | **End** |
+| --- | --- | --- |
+| **Validator Registration** | **12th September** | **6th October** |
+| Pre-Stake Phase | 7th October | 10th November |
+| Validator Activation | 19th November | - |
+
+The steps outlined in this guide are only applicable during the **Validator Registration Phase**.
+
+The **Activation Phase** begins on **November 19**, and we recommend running the activation tool **before** this date to allow your node to download the database early. Learn more about the activation process [here](#what-s-next).
+
+### Prerequisites
+
+- Ensure [Node.js](https://nodejs.org) ≥ v18.20.4 is installed
+- Clone the [Nimiq Validator Registration Tool repository](https://github.com/nimiq/validator-registration-tool) and move to the directory
+- Install Yarn globally with `npm install -g yarn`
+- Install all dependencies by executing `yarn` inside the cloned repository
+
+### Step 1: Generate the Validator Keys
+
+Execute the tool without parameters to generate a validator address, signing key, and voting key:
+
+```bash
+node validator-registration.js
+```
+
+The tool generates fresh keys and stores them into the `validator-keys.json` file in your current directory. The screenshot demonstrates an example output of the script:
+
+<img class="object-contain max-h-[max(80vh,220px)]" src="/assets/images/migration/migration.png" alt="Validator example keys" />
+
+<Callout type='tip'>
+
+Save the private keys securely, especially the validator private key! **There is no recovery mechanism for lost private keys**. Once lost, access to your validator and related NIM may be permanently lost.
+
+</Callout>
+
+For detailed guidance through the scripts and their options, run `node validator-registration.js --help`. This will print out the usage instructions.
+
+**Using your own keys**
+
+You can of course also generate your own validator, signing, and/or voting keys using other methods. To do so, create a `validator-keys.json` file and replace the auto-generated address and keys with your own. Once you have added your keys, continue with the next step.
```

Comment:
```suggestion
You can of course also generate your own validator, signing, and/or voting keys using other methods. To do so, copy the `validator-keys.json` file generated in the previous step and replace the auto-generated address and keys with your own. Once you have added your keys, continue with the next step.
```

Reply:
I don't understand this comment. If I want to generate my keys, I need to run `node validator-registration.js` as well? If the user generates his own keys, it's because he did not executed the previous step, right?

Reply:
Yes. When you clone the repository, there won't be any `validator-keys.json` file to edit. So we have to run the command first to create it.